kodi_addon_checker/check_dependencies.py

```diff
@@ -81,8 +95,12 @@ def _get_users_dependencies(parsed_xml):
     """Gets all the dependencies from a given addon
         :parsed_xml: parsed addon.xml
     """
+    user_deps = {}
+    optional_deps = {}
+    for i in parsed_xml.findall("requires/import"):
+        if i.get("otpional"):
```

Comment:
Typo. After correcting, I get the info report instead of the error when running locally.

Reply:
It was discussed that since this is a check for optional dependencies and an author would be aware of them(they require extra changes in addon.xml) so it would be nice to not make them error/warning but to keep it as info, just a nice suggestion for some improvement in addon.

Reply:
Makes sense, with the PR as is I still get the error. Changing i.get("otpional") to i.get("optional") I get the expected info report.